modules/virt-enabling-disabling-vm-delete-protection-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-cli_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the command line
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the command line. Should you change your mind, you can easily disable delete protection for a VM.
+
+By default, VMs are not enabled with delete protection. You must set the option for each individual VM.
+
+.Prerequisites
+
+* You have installed {VirtProductName} 4.19 or later.
```

Comment:
This isn't necessary because you're only applying this PR to the 4.19+ branch. I'd remove this

Reply:
I'd like to leave the 4.19 mention and remove "or later."

Reply:
This one I would strongly advise against keeping. If you think about it, people would be adding this for all new 4.19 features. It doesn't make sense to do it here, but not everywhere. Unless there's a really good argument for it, I think you should remove it. But feel free to talk to your team for further input!